### PR TITLE
LDO kit example and some clarification

### PR DESCRIPTION
**Pull request by millerlarson**
I hit some confusion on setting up the SB PCB 2pc board for my v2 LDO kit and figured I'd try to clear up where I got lost. Hopefully this example helps illustrate how and where to leverage the AUX header on the toolhead a bit better with regard to the breakout board.

### CODE REVIEW

File: PCBs/Stealthburner_Toolhead_PCB/README.md [hunk 1]
```diff
@@ -88,10 +88,14 @@ to get the LEDs and ERCF filament sensor header functional, you will need to run
...
 ![Here](Images/LDO/Octopus_XES.png)
-the 4 pin wire that is going into the breakout board that come from the XY endstop PCB needs to have its X wire (Blue in the diagram) removed and connected directly to the octopus, This is because the breakout board shares the signal of the X endstop on the toolhead and the gantry, so by removing the X endstop from the breakout and connecting it directly to the octopus you can split that up and use the toolhead header as an auxiliary connection. Once you have done that refer to the diagrams above on how to use it
+The 4 pin wire that is going into the breakout board that comes from the XY endstop PCB needs to have its X wire (Blue in the diagram) removed and connected directly to the octopus as the new x endstop. This is because the breakout board shares the signal of the X endstop on the toolhead and the gantry, so by moving the X endstop wire from the breakout board and connecting it directly to the octopus you can now use the toolhead AUX header as an optional connection. Once you have done that refer to the [diagrams above](#options-for-aux-pin) on how to use it.
```
Comment (hartk1213) [inline PCBs/Stealthburner_Toolhead_PCB/README.md:93]: this sounds good!

**Review by hartk1213 — CHANGES_REQUESTED** [review]
if you can make that single change to keep it as Stock endstop i think its good to go after that

File: PCBs/Stealthburner_Toolhead_PCB/README.md [hunk 2]
```diff
@@ -88,10 +88,14 @@ to get the LEDs and ERCF filament sensor header functional, you will need to run
 ![Here](Images/LDO/Octopus_LED_ERCF.png)
 
 #### For LDO V2 and LDO Trident only (OPTIONAL)
-if you are using the stock XY endstop on the xy joint and you want to use the X endstop header on the toolhead board as an auxiliary port for 3rd fan pin, toolhead mounted chamber thermistor. Then there is 1 thing that will need to be done 
+If you are using the LDO XY endstop on the xy joint and you want to use the AUX header on the toolhead board as an optional port for 3rd fan pin, toolhead mounted chamber thermistor via the XES breakout header, then there is 1 thing that will need to be done
```
[the remark this answers is not in the document]
Comment (millerlarson) [reply PCBs/Stealthburner_Toolhead_PCB/README.md]: Done in [6e4f22c](https://github.com/hartk1213/MISC/pull/46/commits/6e4f22c85e541eafab4471c52c59f51c7ac6b2b8)

**Review by hartk1213 — APPROVED** [review]
looks great!

**Pull request merged**